src/apprunner/explorer/apprunnerNode.ts

```diff
@@ -92,14 +92,14 @@ export class AppRunnerNode extends AWSTreeNodeBase {
 
     private clearPollTimer(): void {
         if (this.pollingNodes.size === 0 && this.pollTimer) {
-            clearInterval(this.pollTimer)
+            awsToolkit.clock.clearInterval(this.pollTimer)
             this.pollTimer = undefined
         }
     }
 
     public startPolling(id: string): void {
         this.pollingNodes.add(id)
-        this.pollTimer = this.pollTimer ?? setInterval(this.refresh.bind(this), POLLING_INTERVAL)
+        this.pollTimer = this.pollTimer ?? awsToolkit.clock.setInterval(this.refresh.bind(this), POLLING_INTERVAL)
```

Comment:
should we have a lint rule to prevent direct usage of the `window.setInterval` / setTimeout / etc?

Reply:
> But using the real thing has no effect if you are not mocking the clock.

Main concern for me is if someone mocks the clock but doesn't use this global wrapper, then has weird effects and doesn't know why. 

> But I consider it relatively low priority.

Not a blocker here. Backlogged: #2347

Reply:
> Main concern for me is if someone mocks the clock but doesn't use this global wrapper, then has weird effects and doesn't know why.

Yup that's the biggest concern. I'll add a big warning for `installFakeClock` for now until linting can be added.